modules/virt-creating-linux-bridge-nncp.adoc

```diff
@@ -3,15 +3,15 @@
 // * virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc
 
 :_content-type: PROCEDURE
-[id="virt-configuring-linux-bridge-nncp_{context}"]
-= Creating a Linux bridge using a node network configuration policy
+[id="virt-creating-linux-bridge-nncp_{context}"]
 
-As a network administrator, you can create a Linux bridge interface on nodes in the cluster by applying a `NodeNetworkConfigurationPolicy` manifest to the cluster.
+= Creating a Linux bridge node network configuration policy
+
+Use a `NodeNetworkConfigurationPolicy` manifest YAML file to create the Linux bridge.
 
 .Procedure
 
-. Create the `NodeNetworkConfigurationPolicy` manifest. This YAML file is an example of a manifest for a Linux bridge interface.
-It includes samples values that you must replace with your own information.
+. Create the `NodeNetworkConfigurationPolicy` manifest. This example includes sample values that you must replace with your own information.
 
```

Comment:
I know this was already like this, but the following code block isn't indented properly with the above step, can you add a + on this line so that it is?

Also, it's a single step procedure, so it really should be an unnumbered bullet and not a number 1. Again, already like that, but if you wouldn't mind fixing now or later :)